modules/translate_bot.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import googletrans
+import telegram
+from telegram import Update, ChatAction
+from telegram.ext import CallbackContext
+
+from modules.abstract_module import AbstractModule
+from utils.decorators import register_module, register_command, send_action, log_errors
+
+
+@register_module()
+class TranslateBot(AbstractModule):
+    @register_command(command="trans", short_desc="Translates the given sentence.",
+                      long_desc="Translates the given sentence and returns the translation by setting a base language, translation language and providing a text. See `/trans languages` for a list of all supported languages and their corresponding key.",
+                      usage=["/trans [lang-base] [lang-trans] [text]", "/trans en de Please translate this to German.", "/trans languages"])
+    @log_errors()
+    @send_action(action=ChatAction.TYPING)
+    def translate(self, update: Update, context: CallbackContext):
+        try:
```

Comment:
Try/catch block shouldn't be needed since this is already covered with the "log_errors" decorator.